src/ci/citool/tests/test-jobs.yml

```diff
@@ -64,11 +64,11 @@ envs:
 # These jobs automatically inherit envs.pr, to avoid repeating
 # it in each job definition.
 pr:
-  - name: mingw-check-1
+  - name: pr-check-1
     <<: *job-linux-4c
-  - name: mingw-check-2
+  - name: pr-check-2
     <<: *job-linux-4c
-  - name: mingw-check-tidy
+  - name: pr-check-tidy
```

Comment:
Maybe just `tidy`? It doesn't do any `x check` build, and it's also executed on `auto` builds, so calling it `pr` is a bit misleading.